src/objects/zcl_abapgit_oo_class.clas.abap

```diff
@@ -770,4 +798,34 @@ CLASS ZCL_ABAPGIT_OO_CLASS IMPLEMENTATION.
     lv_cp = cl_oo_classname_service=>get_classpool_name( iv_class_name ).
     READ TEXTPOOL lv_cp INTO rt_text_pool LANGUAGE iv_language. "#EC CI_READ_REP
   ENDMETHOD.
+
+
+  METHOD is_min_702_sp14_check.
+
+    DATA: lv_patchlevel   TYPE sappatchlv,
+          lv_patchlevel_i TYPE i.
+
+    rv_is_min_702_sp14 = abap_true.
+
+    SELECT SINGLE extrelease
+           FROM cvers
+           INTO ( lv_patchlevel )
+           WHERE component = 'SAP_ABA'
```

Comment:
Shouldn't this check `SAP_BASIS` instead of `SAP_ABA` ? Will otherwise unnecessarily fail on 702 systems without ABA.

Reply:
suggest checking for the problem instead of the release, it was if `CONSTANTS: lc_class_source_extension TYPE sychar02 VALUE 'CS'` did not exist?

Reply:
- @flaiker I've never seen SAP_BASIS without SAP_ABA, and they always had the same value (AFAIK difference only in recent releases e.g. SAP_BASIS 752 - SAP_ABA 75C).
- @larshp The issue is that it's only the kernel which refuses the extension type 'CS', it's not related to ABAP code at all. I don't see other solution than checking the release (or the kernel version, as per note [1505127 - Extension type CS does not exist](https://launchpad.support.sap.com/#/notes/1505127/E), it should be kernel >= 720 SP63 or 72L SP16, but I don't know the rule to do the check with "72L")

Reply:
Well we had this whole decoupling from non SAP_BASIS components thing a while ago to support mainly SAP internal systems if i recall correctly (#1826 #1874).

Reply:
is it possible somehow to get the kernel version number?

Reply:
@flaiker I can do it for SAP_BASIS if you prefer, but it really doesn't matter according to me (my feeling is that a system with SAP_BASIS but without SAP_ABA wouldn't compile).
@larshp Sure, ABAP code behind menu > system > kernel is `CALL 'SAPCORE' ...` If you want it, then should the test be this one: method does_kernel_handle_cs_ext returns true `if ( ver='720' and sp>16 ) or ( ver='72L' and sp>=16 ) or ver>'72'`

Reply:
> my feeling is that a system with SAP_BASIS but without SAP_ABA wouldn't compile

I can only take the word from Jakub in the PRs I mentioned that it works as I don't have access to such systems either. It does not really matter in this case, as it's not a compile time restriction and I doubt the basis systems in question have such a low release.

Reply:
- @flaiker Okay for SAP_BASIS (I scanned all abapGit objects and they use only SAP_BASIS objects; I had a wrong feeling about SAP_ABA being "deeper" than SAP_BASIS but in fact it's described as "cross application component", so SAP_BASIS is deeper than SAP_ABA).
- @larshp Which test of SAP_BASIS versus Kernel is better for you? (note: maybe we could forget "72L", if it's a SAP internal kernel)
+ Looping @filak-sap in case he has an opinion on the proposed correction.